### PR TITLE
Add more chars to conversion table

### PR DESCRIPTION
**Pull request by fmvilas**
Hi again!

I went to https://github.com/dodo/node-slug/blob/master/slug.js and saw they updated/fixed their conversion table. I'm not sure it should replace symbols by default. What do you think?

Cheers!

### CODE REVIEW

**Comment by ggregoire** [comment]
Please no currencies/symbols.

From the README:

> * Doesn't try to do anything fancy with symbols (just removes them)

**Comment by bengourley** [comment]
Thanks @fmvilas and sorry for the delay (again)! As @ggregoire mentioned, I only want to substitute letter-characters, not do anything with currencies/symbols.

If you could provide just the letters that'd be great 😄

File: slugg.js [hunk 1]
```diff
@@ -86,12 +86,46 @@ var chars = slugg.chars = {
...
+  '₯': 'drachma', '₰': 'penny', '₱': 'peso', '₲': 'guarani', '₳': 'austral',
+  '₴': 'hryvnia', '₵': 'cedi', '¢': 'cent', '¥': 'yen', '元': 'yuan',
+  '円': 'yen', '﷼': 'rial', '₠': 'ecu', '¤': 'currency', '฿': 'baht',
+  "$": 'dollar', '₹': 'indian rupee',
+  // Symbols
+  '©':'(c)', 'œ': 'oe', 'Œ': 'OE', '∑': 'sum', '®': '(r)', '†': '+',
+  '“': '"', '”': '"', '‘': "'", '’': "'", '∂': 'd', 'ƒ': 'f', '™': 'tm',
+  '℠': 'sm', '…': '...', '˚': 'o', 'º': 'o', 'ª': 'a', '•': '*',
+  '∆': 'delta', '∞': 'infinity', '♥': 'love', '&': 'and', '|': 'or',
+  '<': 'less', '>': 'greater'
```
Comment (fmvilas) [inline slugg.js:128]:
```suggestion
  '€': '', '₢': '', '₣': '', '£': '',
  '₤': '', '₥': '', '₦': '', '₧': '', '₨': '',
  '₩': '', '₪': '', '₫': '', '₭': '', '₮': '',
  '₯': '', '₰': '', '₱': '', '₲': '', '₳': '',
  '₴': '', '₵': '', '¢': '', '¥': '', '元': '',
  '円': '', '﷼': '', '₠': '', '¤': '', '฿': '',
  "$": '', '₹': '',
  // Symbols
  '©':'', 'œ': '', 'Œ': '', '∑': '', '®': '', '†': '',
  '“': '', '”': '', '‘': '', '’': '', '∂': '', 'ƒ': '', '™': '',
  '℠': '', '…': '', '˚': '', 'º': '', 'ª': '', '•': '',
  '∆': '', '∞': '', '♥': '', '&': '', '|': '',
  '<': '', '>': ''
```

Comment (fmvilas) [reply slugg.js:128]: I don't have access to my fork anymore so I can't do any changes. I hope you can apply the suggestion above.

Better late than never, right? 😄 Feel free to ignore it if it doesn't make sense.

**Pull request closed without merging**